src/cpu/aarch64/jit_primitive_conf.hpp

```diff
@@ -0,0 +1,727 @@
+/*******************************************************************************
+* Copyright 2016-2020 Intel Corporation
+* Copyright 2020 FUJITSU LIMITED
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef CPU_AARCH64_JIT_PRIMITIVE_CONF_HPP
+#define CPU_AARCH64_JIT_PRIMITIVE_CONF_HPP
+
+#include <stdint.h>
+
+#include "common/primitive_attr.hpp"
+#include "cpu/aarch64/cpu_isa_traits.hpp"
+
+namespace dnnl {
+namespace impl {
+namespace cpu {
+namespace aarch64 {
+
+/* convolution */
+enum conv_version_t {
+    ver_unused,
+    ver_fma,
+    ver_avx512_core,
```

Comment:
[here and other places] I was wondering if we should substitute `avx512_core` --> `sve512` [or whichever ISA is more appropriate]

Reply:
Thx! I'll delete `sve512`.